FHEM/00_SIGNALduino.pm

```diff
@@ -671,6 +671,9 @@ sub SIGNALduino_Set_raw {
   my ($hash, @a) = @_;
   $hash->{logMethod}->($hash->{NAME}, 4, "$hash->{NAME}: Set_raw, ".join(' ',@a));
   SIGNALduino_AddSendQueue($hash,$a[1]);
+  if ($a[1] =~ m/^C[D|E]R/) { # enable/disable data reduction
+    SIGNALduino_Get_Command($hash,'config');
```

Comment:
Vielleicht sollten wir das pauschal bei jedem Raw Kommando machen. ?

Reply:
Welches Reading sollte dann z.B. bei diesen Kommandos aktualisiert werden?

```
CED -> Debugausgaben ein
CDD -> Debugausgaben aus
CDL -> LED aus
CEL -> LED ein
CSmscnt=[Wert] -> Wiederholungszähler für den split von MS Nachrichten
CSmuthresh=[Wert] -> Schwellwert für den split von MU Nachrichten (0=aus)
CSmcmbl=[Wert] -> minbitlen für MC-Nachrichten
CSfifolimit=[Wert] -> Schwellwert für debug Ausgabe der Pulsanzahl im FIFO Puffer
```

Reply:
ich dachte eher daran, wenn jemand über diesen Befehl Register oder sonst was ändert.
Von den genannten gibt es eh keinen in der Firmware.

Reply:
> ich dachte eher daran, wenn jemand über diesen Befehl Register oder sonst was ändert.

Mir fällt nichts ein, was aktualisiert werden müsste, wenn Register geändert werden. Die wichtigen Einstellungen, wie z.B. Frequenz, Bandbreite usw. wird ja wahrscheinlich kaum jemand mittels Raw-Befehl ändern, da es dafür ja direkte Befehle gibt. Beim Set darüber werden die entsprechenden Readings ja jetzt schon aktualisiert.

> Von den genannten gibt es eh keinen in der Firmware.

Die zitierten Kommandos stammen aus der commandref. Wollen wir diese dann vielleicht gleich mal bereinigen?

Reply:
Ich gebe dir recht. Es gibt spezielle Set Befehle dafür. Commandref sollten wir dann aktualisieren.

Reply:
Ich habe die in "unserer" Firmware nicht zur Verfügung stehenden Befehle jetzt aus der Commandref entfernt.